components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -180,7 +180,8 @@ export class ReportingComponent implements OnInit, OnDestroy {
     const allUrlParameters$ = this.getAllUrlParameters();
 
     this.endDate$ = this.reportQuery.state.pipe(map((reportQuery: ReportQuery) =>
-      reportQuery.endDate.toDate()));
+      new Date(reportQuery.endDate.year(), reportQuery.endDate.month(),
```

Comment:
Converting the moment to a date object without any time zone conversion. Where before the date's hours would be offset by the user's local timezone.